app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -186,79 +238,30 @@
 }
 
 .gem-c-feedback__option-list {
+  display: inline-block;
   list-style-type: none;
-  margin: 0;
   padding: 0;
 }
 
 
 .js-enabled {
-  .gem-c-feedback__js-prompt-questions {
-    @include govuk-media-query($until: tablet) {
-      display: grid;
-    }
-  }
-
-  .gem-c-feedback__prompt-question {
-    @include govuk-media-query($until: tablet) {
-      grid-area: 1 / 1;
-    }
-  }
-
   .gem-c-feedback__option-list {
     list-style-type: none;
     margin: 0;
     padding: 0;
-
-    @include govuk-media-query($until: tablet) {
-      grid-area: 1 / 1 / 2 / 2;
-      display: grid;
-      grid-template-columns: 140px 1fr;
-      // older grid spec
-      grid-row-gap: govuk-spacing(3);
-      // newer grid spec
-      row-gap: govuk-spacing(3); // sass-lint:disable-line no-misspelled-properties
-    }
+    margin-right: govuk-spacing(2);
   }
 
   .gem-c-feedback__option-list-item {
+    position: relative;
```

Comment:
Am I right in thinking this is to capture the pseudo element on the link? Only the link itself already has position relative, so I'm not sure this is needed?